lab-remil/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-remil",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot"
+  },
```

Comment:
Forgot to leave this here. You had the .eslintrc but forgot to add a script "lint": "./node_modules/eslint/bin/eslint.js .",    add one in and resubmit for full points